solution.kt

```diff
@@ -0,0 +1,61 @@
+class Student(_Fname: String, _Lname: String, _testScore: Int) {
+
+      var Fname: String
+            get() = field.capitalize()
+      var Lname: String
+            get() = field.capitalize()
+      var testScore: Int
+
+
+      init {
+            Fname = _Fname
+            Lname = _Lname
+            testScore = _testScore
+      }
+      fun addTest (name:String , score:Int):  MutableMap <String, Int> {
+            var map = mutableMapOf<String, Int>()
+            if (score >= 0 && score <= 100) {
+                  map.put("$name", score)
+            } else {
+                  println("Enter the score between 0 to 100")
+            }
+            return map
+      }
+      fun display(map: MutableMap<String,Int>){
+
+            var students = map.keys.toMutableList()
+            var scores = map.values.toMutableList()
+            println(students)
+            println(scores)
+
+
+      }
+
+      fun main (){
```

Comment:
The main function is the entry point of the program, so it should not be included in any class.

Reply:
sorry for the mistake i correct it hope you accept my work